### PR TITLE
Create first-interaction.yml

### PR DESCRIPTION
**Pull request by aniruddhaadak80**
When a  new contributor opens an issue, they will receive a personalized welcome message that mentions their username.

Solution for issue #74

### CODE REVIEW

File: .github/workflows/first-interaction.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: Welcome New Contributors
+
+on:
+  issues:
+    types: [opened]
+
```
Comment (jbampton) [inline .github/workflows/first-interaction.yml:6]:
```suggestion
on: [pull_request, issues]

```

**Review by jbampton — CHANGES_REQUESTED** [review]
https://github.com/brisbanesocialchess/.github/actions/runs/11572982256/job/32214112343?pr=76

https://github.com/actions/first-interaction?tab=readme-ov-file#usage

**Review by jbampton — APPROVED** [review]

**Comment by jbampton** [comment]
Thanks !!

**Comment by aniruddhaadak80** [comment]
> Thanks !!

Most welcome @jbampton .I'm very glad that I could help to resolve this issue.

**Pull request merged**